contrib/fwupd.spec.in

```diff
@@ -51,8 +61,13 @@ Release:   0.#BUILD#%{?alphatag}%{?dist}
 License:   LGPL-2.1-or-later
 URL:       https://github.com/fwupd/fwupd
 Source0:   http://people.freedesktop.org/~hughsient/releases/%{name}-%{version}.tar.xz
+%if 0%{?libfwupd_19x_version}
+Source1:   http://people.freedesktop.org/~hughsient/releases/fwupd-efi-1.7.tar.gz
```

Comment:
not https://github.com/fwupd/fwupd-efi/archive/refs/tags/1.7.tar.gz?

Reply:
that's a github-generated tarball not the release tarballs. Maybe an action item for me would be to create proper releases on github for all the old tarballs...

Reply:
What's wrong with the github generated one?  I guess I have no idea why there needs to be an extra in the first place.

Reply:
you're completely right; we've used the github generated ones for the last couple of releases. Fixed.